ctapipe/io/tests/test_datawriter.py

```diff
@@ -1,18 +1,35 @@
 #!/usr/bin/env python3
 
 import numpy as np
-from ctapipe.io.dl1writer import DL1Writer, DL1_DATA_MODEL_VERSION
+from ctapipe.io.datawriter import DataWriter, DATA_MODEL_VERSION
 from ctapipe.utils import get_dataset_path
-from ctapipe.io import EventSource
+from ctapipe.io import EventSource, DataLevel
 from ctapipe.calib import CameraCalibrator
 from pathlib import Path
 from ctapipe.instrument import SubarrayDescription
 from copy import deepcopy
 import tables
 import logging
+from astropy import units as u
 
 
-def test_dl1writer(tmpdir: Path):
+def generate_dummy_dl2(event):
```

Comment:
should be `test_xxx` otherwise pytest doesn't find it (at least this is what I tested by checking out the PR)

Reply:
that is not a test, but a function to generate some dummy data for one of the tests (it just fills in the DL2 containers, since we don't yet have a reconstructor that does it)

Reply:
ah yes for the roundtrip test ok